src/main/java/no/sikt/nva/pubchannels/model/Journal.java

```diff
@@ -14,67 +15,52 @@ public final class Journal implements Immutable {
     private static final String ID_FIELD = "id";
     private static final String NAME_FIELD = "name";
     private static final String IDENTIFIER_FIELD = "identifier";
-    private static final String ELECTRONIC_ISSN_FIELD = "eIssn";
-    private static final String ISSN_FIELD = "issn";
-    private static final String ACTIVE_FIELD = "active";
-    private static final String LANGUAGE_FIELD = "language";
-    private static final String WEBPAGE_FIELD = "webpage";
-    private static final String PUBLISHER_FIELD = "publisher";
-    private static final String NPI_DOMAIN_FIELD = "npiDomain";
-    private static final String SCIENTIFIC_VALUE_FIELD = "scientificValue";
+    private static final String YEAR_FIELD = "year";
+    private static final String ONLINE_ISSN_FIELD = "onlineIssn";
+    private static final String PRINT_ISSN_FIELD = "printIssn";
+    private static final String LEVEL_FIELD = "level";
+    private static final String LANDING_PAGE_FIELD = "landingPage";
 
     @JsonProperty(CONTEXT_FIELD)
     private final URI context;
     @JsonProperty(ID_FIELD)
     private final URI id;
     @JsonProperty(IDENTIFIER_FIELD)
     private final String identifier;
+    @JsonProperty(YEAR_FIELD)
+    private final String year;
     @JsonProperty(NAME_FIELD)
     private final String name;
-    @JsonProperty(ELECTRONIC_ISSN_FIELD)
-    private final String electronicIssn;
-    @JsonProperty(ISSN_FIELD)
-    private final String issn;
-    @JsonProperty(ACTIVE_FIELD)
-    private final boolean active;
-    @JsonProperty(LANGUAGE_FIELD)
-    private final URI language;
-    @JsonProperty(WEBPAGE_FIELD)
-    private final URI webpage;
-    @JsonProperty(PUBLISHER_FIELD)
-    private final URI publisher;
-    @JsonProperty(NPI_DOMAIN_FIELD)
-    private final URI npiDomain;
-    @JsonProperty(SCIENTIFIC_VALUE_FIELD)
-    private final ScientificValue scientificValue;
+    @JsonProperty(ONLINE_ISSN_FIELD)
+    private final String onlineIssn;
+    @JsonProperty(PRINT_ISSN_FIELD)
+    private final String printIssn;
+    @JsonProperty(LEVEL_FIELD)
+    private final String level;
+    @JsonProperty(LANDING_PAGE_FIELD)
+    private final URI landingPage;
 
     @JsonCreator
     @SuppressWarnings("PMD.ExcessiveParameterList")
     public Journal(@JsonProperty(CONTEXT_FIELD) URI context,
                    @JsonProperty(ID_FIELD) URI id,
                    @JsonProperty(IDENTIFIER_FIELD) String identifier,
+                   @JsonProperty(YEAR_FIELD) String year,
```

Comment:
This looks like a dto, the @context and @id fields are irrelevant. I'd create a domain model for the core fields and relegate ownership of @context and @id to the dto